config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-azure.yaml

```diff
@@ -0,0 +1,111 @@
+---
+periodics:
+- interval: 30m
+  name: ci-kubernetes-e2e-azure-scalability
+  cluster: eks-prow-build-cluster
+  decorate: true
+  decoration_config:
+    timeout: 8h
+  path_alias: k8s.io/perf-tests
+  tags:
+  - "perfDashPrefix: azure-100Nodes-master"
+  - "perfDashJobType: performance"
+  - "perfDashBuildsCount: 500"
+  labels:
+    preset-dind-enabled: "true"
+    preset-kind-volume-mounts: "true"
+    preset-capz-containerd-1-7-latest: "true"
+    preset-azure-community: "true"
+  extra_refs:
+  - org: kubernetes
+    repo: perf-tests
+    base_ref: "master"
+    path_alias: "k8s.io/perf-tests"
+  - org: kubernetes-sigs
+    repo: cluster-api-provider-azure
+    base_ref: main # TODO: prow-load template is only on main ATM.
+    path_alias: "sigs.k8s.io/cluster-api-provider-azure"
+    workdir: true
+  spec:
+    serviceAccountName: azure
+    containers:
+    - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20240803-cf1183f2db-master
+      command:
+      - runner.sh
+      - ./scripts/ci-entrypoint.sh
+      args:
+      - bash
+      - -c
+      - >-
+        cd ${GOPATH}/src/k8s.io/perf-tests/ &&
+        ./run-e2e.sh cluster-loader2
+        --nodes=100 \
+        --prometheus-scrape-kubelets=true \
+        --prometheus-scrape-node-exporter \
+        --provider=aks \
+        --testconfig=testing/load/config.yaml \
+        --testconfig=testing/huge-service/config.yaml \
+        --testconfig=testing/access-tokens/config.yaml \
+        --testoverrides=./testing/experiments/enable_restart_count_check.yaml \
+        --testoverrides=./testing/experiments/use_simple_latency_query.yaml \
+        --testoverrides=./testing/overrides/load_throughput.yaml \
+        --v=2
+      securityContext:
+        privileged: true
+      env:
+        # CAPZ variables
+      - name: CLUSTER_TEMPLATE
+        value: "test/ci/cluster-template-prow-load.yaml"
+      - name: NODE_MACHINE_TYPE
+        value: "Standard_D16s_v3"
+      - name: TEST_WINDOWS
+        value: "false"
+      - name: KUBERNETES_VERSION
+        value: "v1.25.3"
```

Comment:
@Jont828 will we run the test with K8s 1.25.3 ??

Reply:
Currently on the perf-dash, they have tests for 1.28, 1.29, 1.30, 1.31, and master/main. I'll see if we can run this off of the master branch.